### PR TITLE
chore: Update README

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,11 @@
 
 <h4 align="center">
   <a href="https://docs.medusajs.com">Documentation</a> |
-  <a href="https://demo.medusajs.com/">Medusa Admin Demo</a> |
   <a href="https://www.medusajs.com">Website</a>
 </h4>
 
 <p align="center">
-An open source composable commerce engine built for developers.
+  Building blocks for digital blocks
```
Comment (shahednasser) [inline README.md:16]: Is it intended to be digital blocks? Sounds weird

File: README.md [hunk 2]
```diff
@@ -37,49 +36,39 @@ An open source composable commerce engine built for developers.
 
 ## Getting Started
 
-Follow our [quickstart guide](https://docs.medusajs.com/quickstart/quick-start) to learn how to set up a Medusa server.
+Visit the [Quickstart Guide](https://docs.medusajs.com/quickstart/quick-start) to set up a server.
 
-### Requirements
-
-You can check out [this documentation for details about setting up your environment](https://docs.medusajs.com/tutorial/set-up-your-development-environment).
+Visit the [Docs](https://docs.medusajs.com/tutorial/set-up-your-development-environment) to learn more about our system requirements.
```
Comment (shahednasser) [inline README.md:41]:
```suggestion
Visit the [Docs](https://docs.medusajs.com/development/backend/prepare-environment) to learn more about our system requirements.
```

File: README.md (hunk 1 above)
Comment (olivermrbl) [reply README.md:16]: Haha no - nice catch! Thanks :)

**Comment by srindom** [comment]
Should we update logo to be BW too

**Review by shahednasser — APPROVED** [review]
LGTM

**Pull request merged**